plugins/modules/proxysql_query_rules.py

```diff
@@ -98,6 +98,18 @@
       - The number of milliseconds for which to cache the result of the query.
         Note in ProxySQL 1.1 I(cache_ttl) was in seconds.
     type: int
+  cache_empty_result:
+    description:
+      - Controls if resultset without rows will be cached or not.
+    type: bool
+  multiplex:
+    description:
+      - If 0, multiplex will be disabled. If 1, multiplex could be re-enabled
+        if there aren't any other conditions preventing this (like user
+        variables or transactions). If 2, multiplexing is not disabled for just
+        the current query.See wiki Default is NULL, thus not modifying
+        multiplexing policies.
+    type: int
```

Comment:
A good suggestion, but the most difficult one will be to find the abbreviation for the last state, i.e. `2'.
- disabled (0)
- enabled (1)
- special (2)

The most important thing is that it doesn't confuse users.

Reply:
maybe `transaction` or `query` ?

Reply:
I would still leave the numeric values, this would at least match the reality of Proxy SQL. This value is actually only applicable to queries starting with @ (as far as I understand). It is difficult to call it a transaction, and if you call it a query, it will confuse users, because in any case, everything that changes here refers to the query.


This is what the Proxy SQL documentation reports from the section on multiplexing.
> Tuning multiplexing
>
> `mysql_query_rules.multiplexing` allows us to enable or disable multiplexing based on matching criteria.
> The field currently accepts these values:
>
> * 0 : disable multiplex
> * 1 : enable multiplex
> * 2 : do not disable multiplex for this specific query containing @